.github/workflows/model_loading.yml

```diff
@@ -0,0 +1,53 @@
+name: Model Loading
+
+on:
+  pull_request:
+    paths:
+      - 'mteb/models/**.py'
```

Comment:
Assuming this indicate "on changes to paths"?

Reply:
Fill to be removed.